torch/csrc/jit/interpreter.cpp

```diff
@@ -596,6 +611,7 @@ struct CodeImpl {
     // guarded input is at index 1
     // the rest of args follow
     emitLoadInputs(node->inputs().slice(1, 1));
+    bi_to_ii_[type_table_.size()] = instructions_.size();
```

Comment:
it is fragile to assume the type_table_ and the number of guards is the same size. Other instructions we add later may add types to the table.